lib/clientBuilder.js

```diff
@@ -166,6 +166,17 @@ proto.verifyRecoveryToken = function (opts) {
   return tx.postToTransaction(this, '/api/v1/authn/recovery/token', opts);
 };
 
+// { resource, (rel), (requestContext)}
+proto.webfinger = function (opts) {
+  var url = '/.well-known/webfinger' + util.toQueryParams(opts);
```

Comment:
We should figure out if a custom auth server issuer is allowed. If so, we can do something like this:

```javascript
var issuer = opts.issuer || sdk.options.issuer || sdk.options.url;
var params = util.omit(opts, 'issuer');
var url = issuer + '/.well-known/webfinger' + util.toQueryParams(params);
```

Reply:
I'm not so sure. @yuliu-okta Can you please comment?

Reply:
spoke to @yuliu-okta offline. He said we may support custom auth server in the future, but not now